cmd/fpga_crihook/main.go

```diff
@@ -34,6 +36,8 @@ const (
 	fpgaRegionEnv          = "FPGA_REGION"
 	fpgaAfuEnv             = "FPGA_AFU"
 	fpgaBitStreamExt       = ".gbs"
+	fpgaDevRegexp          = `\/dev\/intel-fpga-port.(\d)$`
+	aFuIDTemplate          = "/sys/class/fpga/intel-fpga-dev.%s/intel-fpga-port.0/afu_id"
```

Comment:
why capital F in `aFuIDTemplate`?

Reply:
Let's assume there are two FPGA devices. Then there will be dev nodes
```
/dev/intel-fpga-port.0
/dev/intel-fpga-port.1
```
In this case the path to `afu_id` will be `/sys/class/fpga/intel-fpga-dev.1/intel-fpga-port.1/afu_id`, not `/sys/class/fpga/intel-fpga-dev.1/intel-fpga-port.0/afu_id`